dist/windows/installer.nsi

```diff
@@ -119,6 +119,12 @@ Section $(inst_startmenu) ;"Create Start Menu Shortcut"
 
 SectionEnd
 
+Section $(inst_startup) ;"Start qBittorrent on windows start up"
+
+  WriteRegStr HKCU "Software\Microsoft\Windows\CurrentVersion\Run" "qBittorrent" '"$INSTDIR\qbittorrent.exe"'
```

Comment:
IIRC, installation process is performed "as administrator". Are you sure this key will be created in expected place?

Reply:
It works as long as the administrator is current user. But if you run the installation process from a non-admin account and then give admin privilege it will not work.

Reply:
>But if you run the installation process from a non-admin account and then give admin privilege by entering their password, the startup entry will be created for that admin account instead.

So it's unexpected behavior, IMO. I can't approve it.
Waiting for other opinions...

Reply:
We can write to HKLM but that will enable startup for all users. I think this will work for most use cases since most users install softwares from an admin account.

Reply:
>We can write to HKLM but that will enable startup for all users.

We shouldn't do it.
>I think this will work for most use cases since most users install softwares from an admin account.

Sorry, you confuses me. Or maybe I asked the wrong question...
I mean what happens in regular use case when user that has "admin" rights (i.e. the member of Administrators group) performs installation with "admin" rights? E.g. user "John Doe" runs installer and accepts rights elevating in corresponding Windows dialog.
(If someone performs installation as another user explicitly it's expected that key is written for another user.)

Reply:
In that case it will only add for the startup key for that specific user who ran the installer.
It will act the same way as enabling the startup within the qbit settings.

Reply:
Well, it's Ok.
Moreover, as I see we write some other HKCU keys from installer I can suppose it works correctly.